frontend/test/metabase/modes/components/actions/CountByTimeAction.unit.spec.js

```diff
@@ -22,6 +22,6 @@ describe("CountByTimeAction", () => {
       aggregation: [["count"]],
       breakout: [["datetime-field", ["field-id", ORDERS.CREATED_AT.id], "day"]],
     });
-    expect(newCard.display).toEqual("bar");
+    expect(newCard.display).toEqual("line");
```

Comment:
Why did this change?

Reply:
I replaced our use of guessVisualization(card, tableMedata) with question.setDefaultDisplay(), which I guess had slightly different logic, but I think line is more correct for timeseries.